2-completed-app/service/tableClient.js

```diff
@@ -0,0 +1,104 @@
+const { TableClient } = require("@azure/data-tables");
+const env = require("../configure/env")
+const serviceClient = TableClient.fromConnectionString(
+  env.connectionString,
+  env.tableName
+);
+
+const getData = async function(){
```

Comment:
can we have a more meaningful name for this function? What kind of data is it getting? Is it getting all weather information for a specific location? For all locations in a city, state, country? Also adding JSDocs to document the function will be really helpful for users reading the code to get more context about what is being done